plugins/inputs/bigbluebutton/bigbluebutton.go

```diff
@@ -0,0 +1,167 @@
+package bigbluebutton
+
+import (
+	"encoding/xml"
+	"fmt"
+
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/plugins/inputs"
+)
+
+type BigBlueButton struct {
+	URL         string `toml:"url"`
+	APIEndpoint string `toml:"api_endpoint"`
+	SecretKey   string `toml:"secret_key"`
+	ServerName  string `toml:"server_name"`
```

Comment:
Please remove this here and use the telegraf `tags` mechanism to add arbitrary tags.